src/RA_Dlg_AchEditor.cpp

```diff
@@ -875,7 +875,7 @@ INT_PTR Dlg_AchievementEditor::AchievementEditorProc(HWND hDlg, UINT uMsg, WPARA
             RARect winRect;
             GetWindowRect(hDlg, &winRect);
 
-            for (ResizeContent content : vDlgAchEditorResize)
+            for (ResizeContent& content : vDlgAchEditorResize)
                 content.Resize(winRect.Width(), winRect.Height());
```

Comment:
There was no need to make a copy of each element, just modify the element itself through it's reference.